### PR TITLE
docs: Update carousel documentation

### PR DESCRIPTION
**Pull request by xiaotiandada**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: components/carousel/index.zh-CN.md [hunk 1]
```diff
@@ -34,4 +34,6 @@ cover: https://gw.alipayobjects.com/zos/antfincdn/%24C9tmj978R/Carousel.svg
 | next() | 切换到下一面板 |
 | prev() | 切换到上一面板 |
 
+Arrow 问题 可参考 <https://github.com/ant-design/ant-design/issues/12479>
```
Comment (yoyo837) [inline components/carousel/index.zh-CN.md:37]: 英文文档一起补

Comment (xiaotiandada) [reply components/carousel/index.zh-CN.md:37]: done

Comment (xrkffgg) [inline components/carousel/index.zh-CN.md:37]:
```suggestion
Arrow 问题可参考 [#12479](https://github.com/ant-design/ant-design/issues/12479)
```

File: components/carousel/index.en-US.md [hunk 2]
```diff
@@ -33,4 +33,6 @@ A carousel component. Scales with its container.
 | next() | Change current slide to next slide |
 | prev() | Change current slide to previous slide |
 
+Arrow question for reference [issues](https://github.com/ant-design/ant-design/issues/12479)
```
Comment (xrkffgg) [inline components/carousel/index.en-US.md:36]:
```suggestion
Arrow question for reference [#12479](https://github.com/ant-design/ant-design/issues/12479)
```

Comment (xiaotiandada) [reply components/carousel/index.en-US.md]: 谢谢建议 更新晚了一些...

**Pull request merged**